pkg/driver/node.go

```diff
@@ -83,6 +84,7 @@ func newNodeService(driverOptions *Options) nodeService {
 		cloud:         pvsCloud,
 		mounter:       newNodeMounter(),
 		driverOptions: driverOptions,
+		pvmInstanceId: metadata.GetCloudInstanceId(),
```

Comment:
Instead of calling this function twice, wondering if we can just call once before line 78 above and use that in line 78 and 87

Reply:
Called `GetCloudInstanceId` instead of 'GetPVMInstanceId`.
Fixed in the latest commit.